### Import Libraries

In [1]:
import pandas as pd
import numpy as np
from scipy import stats

### Import Data

In [2]:
buyer_ratio=pd.read_csv("BuyerRatio.csv")
buyer_ratio.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

### EDA

In [3]:
buyer_ratio=buyer_ratio.drop('Observed Values',axis=1)

In [4]:
observed=buyer_ratio.to_numpy()
observed

array([[  50,  142,  131,   70],
       [ 435, 1523, 1356,  750]], dtype=int64)

### Hypothesis Test

In [5]:
stat,pvalue,dof,expected=stats.chi2_contingency(observed)

In [6]:
print(stat,pvalue,dof,np.round(expected))

1.595945538661058 0.6603094907091882 3 [[  43.  147.  131.   72.]
 [ 442. 1518. 1356.  748.]]


### Inference

**HYPOTHESIS**

Ho : All proportions are equal

Ha : Not all proportions are equal

Observed = [[50,142,131,70]

[435,1523,1356,750]]

**SOLUTION**

Expected = [[  43.  147.  131.   72.]

 [ 442. 1518. 1356.  748.]]

p value = 0.66, alpha = 0.05

p value is greater than alpha. Hence Ho is failed to reject.

**CONCLUSION**

All proportions are equal.

### End !!!